### ISSUE TITLE
Images not rendering correctly?

### ISSUE THREAD

**Issue by bensoltoff** [uc-cfss/fp-ChihYuChiang#1]
You emailed me about an issue with spaces in links for pictures. Can you point me to a specific example in your project?

**Comment by ChihYuChiang** [comment]
Thanks for responding! I think I solved the problem by cleaning the cache.
On the other hand, since we did our project in my repo, instead of fp-vincentwangpsy, can you kindly update the list (https://uc-cfss.github.io/project_list.html) for us?
https://github.com/uc-cfss/fp-ChihYuChiang/
https://uc-cfss.github.io/fp-ChihYuChiang/index.html

**Comment by bensoltoff** [comment]
Yes, I just noticed that. Just pushed a correction to the class website and it should be live in a few minutes.

**Comment by ChihYuChiang** [comment]
Thank you so much for everything!
